terraform/aws/main.tf

```diff
@@ -240,22 +240,22 @@ resource "random_shuffle" "dns_ips" {
   input        = concat(
     aws_instance.masternode_amd.*.public_ip,
     aws_instance.masternode_arm.*.public_ip,
-    aws_instance.hp_masternode_amd.*.public_ip,
-    aws_instance.hp_masternode_arm.*.public_ip
+      aws_eip.hpmn_arm_eip.*.public_ip,
+      aws_eip.hpmn_amd_eip.*.public_ip
```

Comment:
It's definitely necessary to reference the EIP itself here right? It's not possible to reference the instance instead, and let the graph figure out that the association is necessary before it gets to this step? After all the EIP *is* the `public_ip` associated with the instance.

Reply:
Definitely necessary unfortunately, the public_ip within the instance itself is the one AWS gives it before the EIP, not the EIP.